packages/neutrino/bin/build.js

```diff
@@ -7,10 +7,7 @@ module.exports = (middleware, options) => {
   return build(middleware, options)
     .fork((errors) => {
       spinner.fail('Building project failed');
-      errors.forEach((err) => {
-        console.error(err.stack || err);
-        err.details && console.error(err.details);
-      });
+      errors.forEach(err => console.error(err));
```

Comment:
I agree that just slapping `unary()` right in the `forEach` probably isn't very descriptive, but putting it in utils with a comment should be OK. In fact, adding comments to all our utility functions would be a good thing. ™